README.md

```diff
@@ -33,7 +33,9 @@ $ pnpm run dev
 $ pnpm run build
 ```
 
-:warning: Make sure to use Chromium-based browser for dev mode. Firefox doesn't work due to to [vite's limitation with web worker](https://vitejs.dev/guide/features.html#web-workers).
+:warning: Firefox doesn't work by default due to to [vite's limitation with web worker](https://vitejs.dev/guide/features.html#web-workers).
```

Comment:
Can we say Firefox doesn't work in development (`pnpm run dev`) but does if you do `pnpm run build`? This reads like Firefox doesn't work at all.